### PR TITLE
Add test cases

### PR DESCRIPTION
**Pull request by seanjyjy**
Pr overview

* Add test case for some UI classes (non-gui)
* Add test case for `view manager` & `tab Manager`
* Add test case for `logic manager`
* Add test case for `commandutil` and `commandresult`

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/util/CommandUtil.java [hunk 1]
```diff
@@ -62,6 +65,7 @@ public static ProfileItem getProfileItem(Model model, Index profileItemIndex) th
      * @return Feedback message of the operation result for display.
      */
     public static CommandResult getCommandResult(Model model, String message, TabName tabName) {
+        assert model != null && message != null && tabName != null;
```
Comment (keanecjy) [inline src/main/java/seedu/address/logic/commands/util/CommandUtil.java]: I think requireAllNonNull would be better here

File: src/main/java/seedu/address/logic/commands/util/CommandUtil.java [hunk 2]
```diff
@@ -84,6 +88,8 @@ public static CommandResult getCommandResult(Model model, String message, TabNam
     public static CommandResult getCommandResult(Model model, String message, TabName currentTabName,
         TabName changedTabName, Index index) {
 
+        assert model != null && message != null && currentTabName != null && changedTabName != null && index != null;
```
Comment (keanecjy) [inline src/main/java/seedu/address/logic/commands/util/CommandUtil.java]: Same here, I think requireAllNonNull would be better

**Review by keanecjy — CHANGES_REQUESTED** [review]
Great work, just some nits to fix before it is good to merge.

File: src/main/java/seedu/address/logic/commands/util/CommandUtil.java (hunk 1 above)
Comment (seanjyjy) [reply src/main/java/seedu/address/logic/commands/util/CommandUtil.java]: Thanks for the advice, did not know there is such a thing.

**Review by keanecjy — APPROVED** [review]

**Pull request merged**